src/CmsKitDemo/CmsKitDemoModule.cs

```diff
@@ -321,6 +326,37 @@ private void ConfigureEfCore(ServiceConfigurationContext context)
         });
     }
 
+    private void ConfigureRazorPages()
+    {
+        Configure<RazorPagesOptions>(options =>
+        {
+            options.Conventions.AddPageRoute("/Gallery/Index", "image-gallery");
+            options.Conventions.AddPageRoute("/Gallery/Detail", "image-gallery/detail/{ImageId}");
```

Comment:
I think something like `image-gallery/{id}` or `image-gallery/{id}/detail` is much better convention for understanding

Reply:
Agree, I'll update the page route ✅